packages/expo/src/errors/EntryNotFound.tsx

```diff
@@ -0,0 +1,33 @@
+import { StyleSheet, Text, View } from 'react-native';
+
+export function EntryNotFound() {
+  return (
+    <View style={styles.container}>
+      <Text style={styles.errorTitle}>Application Not Found</Text>
+      <Text style={styles.errorDescription}>
+        The application entrypoint "main" was not registered.
+      </Text>
+    </View>
+  );
+}
```

Comment:
We need to work on wording here – make it consistent and more clear. Currently we have:
- Entry not found
- Application not found
- Application entrypoint not registered

What are the differences between them? It's all the same. Maybe we can do something like

```suggestion
export function AppEntryNotFound() {
  return (
    <View style={styles.container}>
      <Text style={styles.errorTitle}>App entry not found</Text>
      <Text style={styles.errorDescription}>
        The app entry point "main" was not registered. It might be caused by an uncaught error thrown from the top-level code. See the logs for more details.
      </Text>
    </View>
  );
}
```